tests/set/set5.c

```diff
@@ -36,13 +36,27 @@ int main(void) {
 	struct set *s = set_create(NULL, cmp_int);
 	int a[3] = {1200,2400,3600};
 	size_t i;
+	int **plist;
+
+	plist = malloc(5000 * sizeof *plist);
```

Comment:
Prefer `calloc` for array allocations.

Reply:
I disagree quite strongly with this preference.  Sometimes you want `calloc` and sometimes you don't want to initialize all bytes to zero.

In any case, the argument is a compile-time constant and the compiler should warn about overflow.